tests/utils.go

```diff
@@ -478,7 +478,7 @@ func WaitForAllPodsReady(timeout time.Duration, listOptions metav1.ListOptions)
 		virtClient, err := kubecli.GetKubevirtClient()
 		PanicOnError(err)
 
-		podsList, err := virtClient.CoreV1().Pods(k8sv1.NamespaceAll).List(listOptions)
+		podsList, err := virtClient.CoreV1().Pods(KubeVirtInstallNamespace).List(listOptions)
```

Comment:
I think this change impacts the name of the function - you're no longer "waiting for all pods to be ready", and as such should update the function name.

IMHO, if you preserve the name, you should pass the namespace as an argument to the function.

Reply:
+1

Reply:
+1